DEV-11/Transliterator.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Transliteration
+{
+  // Class contains methods
+  // that performs the process of transliteration
+  // from Cyrillic to Latin and back.
+  // According to the rules of transliteration,
+  // class dictionaries are compiled
+  public class Transliterator
+  {
+    // Process of transliteration from Cyrillic to Latin
+    public string FromCyrillicToLatin(string text)
+    {
+      StringBuilder sb = new StringBuilder();
+
+      string[] words = SplitIntoTokens(text);
+      foreach (var word in words)
+      {
+        string outputWord = word;
+        foreach (var value in CyrillicLatinDictionary.Keys)
+        {
+          if (outputWord.Contains(value))
+          {
+            outputWord = outputWord.Replace(value, CyrillicLatinDictionary[value]);
+          }
+        }
+        sb.Append(outputWord);
+        sb.Append(" ");
+      }
+      return sb.ToString();
+    }
+
+    // Process of transliteration from Latin to Cyrillic
+    public string FromLatinToCyrillic(string text)
```

Comment:
Are sure that it is impossible to avoid code dublicating? the method is too similar to FromCyrillicToLatin